strax/context.py

```diff
@@ -1152,6 +1153,16 @@ def is_stored(self, run_id, target, **kwargs):
                 continue
         return False
 
+    def _check_forbidden(self):
+        """Check that the forbid_creaction_of config is of tuple type.
+        Otherwise, try to make it a tuple"""
+        forbid_creation_of = self.context_config['forbid_creation_of']
+        if not isinstance(forbid_creation_of, tuple):
+            if not isinstance(forbid_creation_of, str):
+                raise ValueError(f'forbid_creation_of should be of type tuple not'
+                                 f'{type(forbid_creation_of)}')
+            self.context_config['forbid_creation_of'] = ((forbid_creation_of), )
```

Comment:
Sorry to be the bad guy here, but I think we should use https://github.com/AxFoundation/strax/blob/16650cc50f8fa3dd39ef28621e95051656fc644c/strax/utils.py#L226 instead. 

This would make the ValueError obsolete as well.

Reply:
Superb, that is much cleaner! Thanks Daniel! I knew that there was a function somewhere but I thought it was some ty.Union-function. 

I'll leave it in the _check_forbidden to have a traceback that is very clear where one would be wrong.